cs4360/libs/crm/shared/src/lib/contracts/user/user.create.contract.ts

```diff
@@ -0,0 +1,15 @@
+import {ContactDTO, UserDTO, UserTypeDTO} from "@crm/shared";
+
+
+// tslint:disable-next-line:no-empty-interface
+export class CreateUserRequest implements UserDTO {
+  contact: ContactDTO;
+  email: string;
+  id: string;
+  passwordHash: string;
+  userType: UserTypeDTO;
+}
+
+export class CreateUserResponse {
+  id: string;
+}
```

Comment:
@joliver2 @vinsonchin This is what I had in mind for the create users contract. See how these classes are used in the user controller.

These need to be created for both user & contact, for actions: create, findAll, findById, update, and delete.